src/components/Books.js

```diff
@@ -0,0 +1,50 @@
+import { useDispatch, useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { FaBible } from 'react-icons/fa';
+import { authenticate, getBooks } from '../redux/books/books';
+// import bible from '../img/bible.jpg';
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.